package.json

```diff
@@ -27,10 +27,17 @@
   },
   "homepage": "https://github.com/VLCTechHub/chrome-addon#readme",
   "devDependencies": {
-    "sinon-chrome": "0.2.0",
-    "mocha": "5.2.0",
-    "chai": "1.9.2",
-    "phantomjs-prebuilt": "2.1.16",
-    "sinon": "1.11.1"
+    "chai": "^1.9.2",
```

Comment:
out of curiosity, why are they not frozen versions?

![frozen](https://media.tenor.co/images/3d825d31a5c3bbd2a7268549d49c4aca/tenor.gif)